static/collection.js

```diff
@@ -1,47 +1,74 @@
 (async () => {
-    function clear_all_games_table(){
+    function clear_table(){
         const table = document.getElementById('table');
         table.innerText = "";
     }
 
-async function create_all_games_table() {
-    clear_all_games_table();
-    const user = document.getElementById("user").value;
-    fetch('/all_games?user=' + user)
-        .then(async response => {
-            if (response.ok) {
-                const games = await response.json();
-                const table = document.getElementById('table')
-                games.forEach(addGamesToTable)
+    async function create_table(games){
+        const table = document.getElementById('table')
+        const header = table.createTHead();
+        const row = header.insertRow(0);
+        const cell_one = row.insertCell(0);
+        cell_one.innerText = "Game Name (released year)"
+        const cell_two = row.insertCell(1);
+        cell_two.innerText = "Overall Rank"
+        const cell_three = row.insertCell(2);
+        cell_three.innerText = "Average Rating"
+        const cell_four = row.insertCell(3);
+        cell_four.innerText = "Complexity"
+        const cell_five = row.insertCell(4);
+        cell_five.innerText = "Players"
+        games.forEach(addGamesToTable)
 
-                function addGamesToTable(game, index){
-                    const row = table.insertRow(index);
-                    const cell_one = row.insertCell(0);
-                    cell_one.innerText = `${game.name} (${game.yearpublished})`
-                    const cell_two = row.insertCell(1);
-                    cell_two.innerText = `${game.minplayers} - ${game.maxplayers} players`
-                }
+        function addGamesToTable(game, index){
+            const row = table.insertRow(index + 1);
+            const cell_one = row.insertCell(0);
+            cell_one.innerText = `${game.name} (${game.yearpublished})`
+            const cell_two = row.insertCell(1);
+            cell_two.innerText = `${game.overallrank}`
+            const cell_three = row.insertCell(2);
+            cell_three.innerText = `${game.averagerating}`
+            const cell_four = row.insertCell(3);
+            cell_four.innerText = `${game.complexity}`
+            const cell_five = row.insertCell(4);
+            cell_five.innerText = `${game.minplayers} - ${game.maxplayers} players`
+        }
+    }
 
-            } else {
-                const error_field = document.getElementById("game-error");
-                error_field.innerText = await response.text();
-            }
-        });
-}
-async function last_username(){
-      fetch('last_username')
+    async function get_game_collection(order_by) {
```

Comment:
I tried to keep all naming convention of methods and keys and variables as clear and readable as possible.  This is the only one I'm unsure of.  Maybe it should be get_ordered_game_collection?

Reply:
I would say thats personal preference but in js/ts world we mostly do title case. Again nothing wrong with the underscore but typically thats more a python world.

Reply:
Followed this convention!